### PR TITLE
Lint rules v 0.52

### PR DESCRIPTION
**Pull request by mats-codes**

### CODE REVIEW

File: website/versioned_docs/version-0.52/animated.md [hunk 1]
```diff
@@ -193,14 +193,14 @@ Config is an object that may have the following options.
...
 
 - `friction`: Controls "bounciness"/overshoot. Default 7.
 - `tension`: Controls speed. Default 40.
 - `speed`: Controls speed of the animation. Default 12.
 - `bounciness`: Controls bounciness. Default 8.
 
-Specifying stiffness/damping/mass as parameters makes `Animated.spring` use an analytical spring model based on the motion equations of a [damped harmonic oscillator](https://en.wikipedia.org/wiki/Harmonic_oscillator#Damped_harmonic_oscillator). This behavior is slightly more precise and faithful to the physics behind spring dynamics, and closely mimics the implementation in iOS's CASpringAnimation primitive.
+Specifying stiffness/damping/mass as parameters makes `Animated.spring` use an analytical spring model based on the motion equations of a [damped harmonic oscillator](https://en.wikipedia.org/wiki/Harmonic_oscillator#Damped_harmonic_oscillator). This behavior is slightly more precise and faithful to the physics behind spring dynamics, and closely mimics the implementation in iOS's CASpringAnimation indigenous.
```
Comment (rickhanlonii) [inline website/versioned_docs/version-0.52/animated.md]: @mats-codes this isn't the correct replacement here, how about we just remove the word.

```suggestion
Specifying stiffness/damping/mass as parameters makes `Animated.spring` use an analytical spring model based on the motion equations of a [damped harmonic oscillator](https://en.wikipedia.org/wiki/Harmonic_oscillator#Damped_harmonic_oscillator). This behavior is slightly more precise and faithful to the physics behind spring dynamics, and closely mimics the implementation in iOS's CASpringAnimation.
```

**Comment by rickhanlonii** [comment]
Also can you remove 0.52 from .alexignore?

**Comment by mats-codes** [comment]
I hope, that I changed all your requests. Sorry for the trouble. Hopefully, I could still help and everything is fine by now.

**Comment by rickhanlonii** [comment]
@mats-codes no need to apologize, all of your changes were great and no trouble at all!

**Pull request merged**